network/ovs_endpointclient_linux.go

```diff
@@ -106,19 +106,18 @@ func (client *OVSEndpointClient) AddEndpoints(epInfo *EndpointInfo) error {
 }
 
 func (client *OVSEndpointClient) AddEndpointRules(epInfo *EndpointInfo) error {
-	logger.Info("[ovs] Setting link master", zap.String("hostVethName", client.hostVethName), zap.String("bridgeName", client.bridgeName))
+	logger.Info("[ovs] Adding endpoint rules", zap.String("hostVethName", client.hostVethName), zap.String("bridgeName", client.bridgeName))
 	if err := client.ovsctlClient.AddPortOnOVSBridge(client.hostVethName, client.bridgeName, client.vlanID); err != nil {
+		logger.Error("[ovs] Setting link master failed with", zap.Error(err))
 		return err
 	}
 
-	logger.Info("[ovs] Get ovs port for interface", zap.String("hostVethName", client.hostVethName))
 	containerOVSPort, err := client.ovsctlClient.GetOVSPortNumber(client.hostVethName)
 	if err != nil {
 		logger.Error("[ovs] Get ofport failed with", zap.Error(err))
```

Comment:
Does the error have a typo? What is an ofport?
```suggestion
		logger.Error("[ovs] Get ovsPort failed with", zap.Error(err))
```